File: bandlog-api/src/main/java/net/effize/bandlog/domain/user/exception/EmailMalformedException.java
package net.effize.bandlog.domain.user.exception;

import net.effize.bandlog.domain.common.exception.BandlogException;
import net.effize.bandlog.domain.common.exception.ErrorCode;

public class EmailMalformedException extends BandlogException {
    public EmailMalformedException() {
        super(ErrorCode.of("EMAIL_MALFORMED_ERROR"), "이메일 형식이 잘못되었습니다.", "email malformed", 400);
    }
}

File: src/main/kotlin/net/effize/bandlog/user/domain/exception/EmailMalformedException.kt
package net.effize.bandlog.user.domain.exception

import net.effize.bandlog.common.exception.BandlogException
import net.effize.bandlog.common.exception.ErrorCode

class EmailMalformedException : BandlogException(
    errorCode = ErrorCode.of("EMAIL_MALFORMED_ERROR"),
    userMessage = "이메일 형식이 잘못되었습니다.",
    logMessage = "email malformed",
    httpStatusCode = 400
)
